src/util/psbt/serialize.rs

```diff
@@ -172,7 +172,7 @@ impl Deserialize for Vec<u8> {
 
 impl Serialize for PsbtSigHashType {
     fn serialize(&self) -> Vec<u8> {
-        serialize(&self.inner())
+        serialize(&self.inner)
```

Comment:
I prefer to call methods so that if the representation changes it's easier to keep it in sync. (Not that it'd be likely here but may be worth reinforcing good habits.)

Reply:
oh yeah? I was unsure of this and explicitly thought about it so I'm glad you mentioned it. I thought because we were _inside_ the module to use the field not the public function. You're suggesting to _always_ use the function if one exists. Lesson learned, thanks.